parsec/backend/memory/realm.py

```diff
@@ -80,6 +80,11 @@ def is_archived(self) -> bool:
             return False
         return self.last_archiving_configuration_request.configuration.is_archived()
 
+    def is_deletion_planned(self) -> bool:
```

Comment:
add a comment to precise deleted is *also* deletion planned !

Reply:
done